.github/workflows/python-release.yml

```diff
@@ -1,31 +1,52 @@
 name: Python Publish to PYPI
 on:
+  # Automatic release based on either creating a published release or
+  # publishing a draft release (creating a draft release will not trigger the
+  # "release" event type according to the docs).
+  # The workflow file version used is that from the tag corresponding to the
+  # release. This is why the "workflow_dispatch" trigger is configured as well:
```

Comment:
the workflow file contents at the `looker_sdk-v21.0.0` tag has the testpypi url so I can't cause it to publish to prod just by tweaking the release.

Adding this `workflow_dispatch` will hopefully solve this one time issue (though it may come up again in the future for some reason)